hawkbit-simple-ui/src/main/java/org/eclipse/hawkbit/ui/simple/view/TargetView.java

```diff
@@ -967,23 +1122,34 @@ private Icon targetUpdateStatusMapper(final String targetUpdateStatus) {
                 default -> "blue";
             };
 
-            final Icon statusIcon = Utils.tooltip(icon.create(), targetUpdateStatus);
+            final Icon statusIcon = Utils.tooltip(icon.create(), targetUpdateStatus.replace("_"," "));
             statusIcon.setColor(color);
             statusIcon.addClassNames(LumoUtility.IconSize.SMALL);
             return statusIcon;
         }
+    }
 
-        private Icon pollStatusIconMapper(MgmtPollStatus pollStatus) {
-            final Icon pollIcon;
-            if (pollStatus == null) {
-                pollIcon = Utils.tooltip(VaadinIcon.QUESTION_CIRCLE.create(), "No Poll Status");
-            } else if (pollStatus.isOverdue()) {
-                pollIcon = Utils.tooltip(VaadinIcon.EXCLAMATION_CIRCLE.create(), "Overdue");
-            } else {
-                pollIcon = Utils.tooltip(VaadinIcon.CLOCK.create(), "In Time");
-            }
-            pollIcon.addClassNames(LumoUtility.IconSize.SMALL);
-            return pollIcon;
+    // todo change /targets api to reduce api calls ?
+    @EqualsAndHashCode(callSuper = true)
+    public static class MgmtTargetVersioned extends MgmtTarget{
```

Comment:
Maybe TargetWithDs ?
1. MgmtSomething pattern is used for management rest classes and is little bit confusing here
2. Target version is not used as a term in hawkBit , and actually this is the installed DS